frontend/src/pages/Login.tsx

```diff
@@ -14,7 +14,11 @@ import Alert from "@material-ui/lab/Alert";
 
 // Project resources
 import { TwoColumnLayout } from "../components/Layouts";
-import img from "../images/Naamloze-presentatie.jpeg";
+import img from "../images/image 31.png";
```

Comment:
I'd give this image a better name, try to avoid spaces in filenames at least, and try to have a descriptive name